store_app/views/movie_edit.py

```diff
@@ -0,0 +1,75 @@
+from flask import make_response, jsonify, request, g
+from business import connect_external_API, movie_edit_funcs
+from app import app
+
+
+@app.route('/movies', methods=['POST', 'DELETE'])
+def add_or_remove_movie():
+    if request.method == "POST":
+        try:
+            if g.user.role != 'admin':
+                raise ValueError('User must be an admin to perform this function.')
```

Comment:
This is an excellent candidate for a decorator function with which you can "guard" endpoints that require admin privileges to run. See more about decorators here: https://www.programiz.com/python-programming/decorator

Reply:
I will check this out. Thanks a lot!